public/js/utils/app-json-formatter.js

```
@@ -8,48 +8,109 @@ const JsonldUtils = require("./jsonld-utils");
  */
 class AppJsonFormatter {
 
-  static createAccountData(resourceBaseUrl, accountName, accountLabel, accountStatus, accountImage) {
+  static async createAccountGraphs(uri, name, label, img, secretaries, status) {
+    var name = UriUtils.uriToName(uri);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'name'. Consider to delete it or rename it.

'name' is defined here:


(lint/suspicious/noRedeclare)